test/parallel/test-internal-errors.js

```diff
@@ -118,3 +124,8 @@ assert.throws(() => {
                            type: TypeError,
                            message: /^Error for testing 2/ }));
 }, /AssertionError: .+ does not match \S/);
+
+assert.doesNotThrow(() => errors.E('TEST_ERROR_USED_SYMBOL'));
+assert.throws(() => errors.E('TEST_ERROR_USED_SYMBOL'),
```

Comment:
FWIW you could move the arguments indented on the following lines to avoid `new RegExp()` and string concatenation:

```js
assert.throws(
  () => errors.E('TEST_ERROR_USED_SYMBOL'),
  /^AssertionError: Error symbol: TEST_ERROR_USED_SYMBOL was used\.$/);
```